documentation/modules/backup-cli.adoc

```diff
@@ -0,0 +1,1049 @@
+<5> Specify the name of the {osp} region.
+<6> Enter the CA certificate for connecting to the source environment. The certificate is not used when `insecureSkipVerify` is set to `<true>`.
+<7> Specify the API end point URL, for example, `https://<identity_service>/v3`.
+endif::[]
+
+ifdef::cnv[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
+  labels:
+    createdForProviderType: openshift
+    createdForResourceType: providers
+type: Opaque
+stringData:
+  user: <user> <2>
+  password: <password> <3>
+  insecureSkipVerify: <true/false> <4>
+  url: <api_end_point> <5>
+EOF
+----
+<1> The `ownerReferences` section is optional.
+<2> Specify the {virt} user.
+<3> Specify the user password.
+<4> Specify `<true>` to skip certificate verification, which proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed. Specifying `<false>` verifies the certificate.
+<5> Specify the API end point URL.
+endif::[]
+
+. Create a `Provider` manifest for the source provider:
++
+ifdef::vmware[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: Provider
+metadata:
+  name: <source_provider>
+  namespace: <namespace>
+spec:
+  type: vmware
+  url: <api_end_point> <1>
+  settings:
+    vddkInitImage: <registry_route_or_server_path>/vddk:<tag> <2>
+  secret:
+    name: <secret> <3>
+    namespace: <namespace>
+EOF
+----
+<1> Specify the API end point URL, for example, `https://<vCenter_host>/sdk`.
+<2> Specify the VDDK image that you created.
+<3> Specify the name of provider `Secret` CR.
+endif::[]
+
+ifdef::rhv[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: Provider
+metadata:
+  name: <source_provider>
+  namespace: <namespace>
+spec:
+  type: ovirt
+  url: <api_end_point> <1>
+  secret:
+    name: <secret> <2>
+    namespace: <namespace>
+EOF
+----
+<1> Specify the API end point URL, for example, `https://<engine_host>/ovirt-engine/api`.
+<2> Specify the name of provider `Secret` CR.
+endif::[]
+
+ifdef::ova[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: Provider
+metadata:
+  name: <source_provider>
+  namespace: <namespace>
+spec:
+  type: ova
+  url: <api_end_point> <1>
```

Comment:
OVA doesn't have an  `api end point` managed by the user, the value here should be the NFS export, similar to the secrete.

Reply:
This file is going to be discarded. I made the change in new-migrating-virtual-machines-cli.adoc.

Reply:
> OVA doesn't have an `api end point` managed by the user, the value here should be the NFS export, similar to the secrete.

good point